website/source/docs/agent/checks.html.md

```diff
@@ -74,15 +76,17 @@ There are five different kinds of checks:
   valid through the end of the TTL from the time of the last check.
 
 * Docker + Interval - These checks depend on invoking an external application which
-is packaged within a Docker Container. The application is triggered within the running
-container via the Docker Exec API. We expect that the Consul agent user has access
-to either the Docker HTTP API or the unix socket. Consul uses ```$DOCKER_HOST``` to
-determine the Docker API endpoint. The application is expected to run, perform a health
-check of the service running inside the container, and exit with an appropriate exit code.
-The check should be paired with an invocation interval. The shell on which the check
-has to be performed is configurable which makes it possible to run containers which
-have different shells on the same host. Check output for Docker is limited to
-4K. Any output larger than this will be truncated.
+  is packaged within a Docker Container. The application is triggered within the running
+  container via the Docker Exec API. We expect that the Consul agent user has access
+  to either the Docker HTTP API or the unix socket. Consul uses ```$DOCKER_HOST``` to
+  determine the Docker API endpoint. The application is expected to run, perform a health
+  check of the service running inside the container, and exit with an appropriate exit code.
+  The check should be paired with an invocation interval. The shell on which the check
+  has to be performed is configurable which makes it possible to run containers which
+  have different shells on the same host. Check output for Docker is limited to
+  4K. Any output larger than this will be truncated. In Consul 0.9.0 and later, the agent
+  must be configured with [`check_enable_exec`](/docs/agent/options.html#_check_enable_exec)
+  set to `true` in order to enable script checks.
```

Comment:
Is there any other way to perform (other than a script check) to perform this Docker check? If not, then the last sentence should be `to enable Docker health checks`.